app/src/pages/Home/Join.tsx

```diff
@@ -81,7 +87,7 @@ function Join(props: { onBack: () => void }) {
         </Grid>
         <Grid container item>
           <Button size="small" onClick={props.onBack}>
-            Back
+            {t("common:backButton", "Back")}
```

Comment:
any difference here between dot vs colon?

Reply:
> consolidate all of the keys in one namespace

Updated in 57e2225b73747a7aab4af2c7a6c160f6c24a8e1a.